components/menu-item/style.scss

```diff
@@ -27,6 +28,14 @@
 	&:focus:not(:disabled):not([aria-disabled="true"]) {
 		@include menu-style__focus;
 	}
+
+	// Forcefully colorize plugin icons to ensure contrast and cohesion.
+	svg,
+	g,
+	path {
```

Comment:
The choice of elements to target here seems ad-hoc and incomplete, and I worry it could result in some abominations of icons where only parts are recolored (and not other parts such as `circle`, `ellipsis`, `line`, etc).

Example: https://codepen.io/aduth/pen/qKyxXJ

Would a wildcard `*` descendent selector be better?

Reply:
I dislike using !important as much as the next one. But most SVGs use inline styles for strokes and colors, and if we don't use !Iimportant, it means that the icon will have whatever color the plugin author provides, _including_ when the icon is toggled, where it should be white on the gray toggle background. In other words, if an SVG has `fill: green;` as an inline style, it will be green in every single context, hover, toggled, active etc.

Reply:
As an example:

![screen shot 2018-06-25 at 15 47 37](https://user-images.githubusercontent.com/1204802/41853877-2303ddf0-788f-11e8-8f9e-4650a6fa783e.png)

That's when I remove the !important from the is-toggled CSS class.

Reply:
@jasmussen I thought I had styles for that in the plugin itself (white path when toggled :P )

Reply:
As a "plugin developer" I think I agree with Gutenberg forcing a white path when toggled but I also think it shouldn't force a gray color when untoggled. Plugins should only care about providing the right colors for the default state (untoggled).

Reply:
I'd be okay with forcing colors when toggled.

Reply:
To clarify, I have no issue with forcing the color, just that there are more colorable SVG elements than just `svg`, `g`, and `path`.

Reply:
@jasmussen sorry I missed the follow ups here. I agree with forcing on toggle state, but I think e should leave the colors the plugin chose for the normal state.